packages/services/src/models/notification-scanners/SubscriptionReminder.js

```diff
@@ -0,0 +1,34 @@
+import BaseModel from './BaseModel'
+import { User } from '@aragonone/court-backend-server/build/models/objection'
+
+class SubscriptionReminder extends BaseModel {
+  async scan() {
+    let notifications = []
+    const users = await User.query().where({emailVerified: false}).withGraphFetched('[email,emailVerificationToken]')
```

Comment:
we can defined a function in the base model like `getList` or `getUsers` for the queries